packages/services/api/src/modules/organization/resolvers/Organization.ts

```diff
@@ -109,4 +114,47 @@ export const Organization: Pick<
     );
   },
   cleanId: organization => organization.slug,
+  auditLogs: async (organization, arg, ctx) => {
+    const isOwner = ctx.injector.get(AuthManager).ensureOrganizationOwnership({
+      organization: organization.id,
+    });
+    if (!isOwner) {
+      throw new GraphQLError('Unauthorized: You are not authorized to perform this action');
+    }
```

Comment:
TODO @TuvalSimha please move this authentication check into the `AuditLogManager` class method (`getPaginatedAuditLogs`).